samples/nrf9160/nrf_cloud_mqtt_multi_service/tests/application/prj.conf

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# ZTEST
+CONFIG_ZTEST=y
+CONFIG_ZTEST_STACK_SIZE=4096
+CONFIG_CJSON_LIB=y
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+
+## May or may not be needed
+#CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL_INF=y
+#CONFIG_NRF_CLOUD_MQTT=y
+#CONFIG_NETWORKING=y #Needed by MQTT
+
+
+## Config dependencies of application.c
+#CONFIG_NRF_MODEM_LIB=y
```

Comment:
Can you clean some of this config file up?

Reply:
Absolutely